package.json

```diff
@@ -8,8 +8,10 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "moment": "^2.29.1",
```

Comment:
There are better alternatives to moment, even the authors suggest looking for something else: https://momentjs.com/docs/#/-project-status/

Reply:
Yeah, it's def lightweight - moment is actually a MASSIVE dependency